docs/gathered-data.md

```diff
@@ -13,1040 +13,1878 @@ archives are packaged as `.tar.gz` files in `/var/lib/insights-operator`.
-* Id in config: clusterconfig/openshift_authentication_logs
-* Since versions:
-  - 4.7+
+### Backported versions
+None
+
+### Changes
+None
 
 
 ## OpenshiftLogging
 
-collects `logging.openshift.io` resources
+Collects `logging.openshift.io` resources
 from Openshift Logging Stack.
 
-API Reference:
+### API Reference
+- https://github.com/openshift/cluster-logging-operator/blob/master/pkg/apis/logging/v1/clusterlogging_types.go
+
+### Sample data
+- [docs/insights-archive-sample/config/logging/openshift-logging/instance.json](./insights-archive-sample/config/logging/openshift-logging/instance.json)
+
+### Location in archive
+- `config/logging/{namespace}/{name}.json`
+
+### Config ID
+`clusterconfig/openshift_logging`
 
-	https://github.com/openshift/cluster-logging-operator/blob/master/pkg/apis/logging/v1/clusterlogging_types.go
+### Released version
+- 4.9.0
 
-* Location in archive: config/logging/{namespace}/{name}.json
-* Id in config: clusterconfig/openshift_logging
-* Since versions:
-  - 4.9+
+### Backported versions
+None
+
+### Changes
+None
 
 
 ## OpenshiftMachineAPIEvents
 
-collects warning ("abnormal") events
-from "openshift-machine-api" namespace
+Collects warning ("abnormal") events
+from `openshift-machine-api` namespace
+
+### API Reference
+None
+
+### Sample data
+- [docs/insights-archive-sample/events/openshift-machine-api.json](./insights-archive-sample/events/openshift-machine-api.json)
+
+### Location in archive
+- `events/openshift-machine-api.json`
+
+### Config ID
+`clusterconfig/openshift_machine_api_events`
+
+### Released version
+- 4.12.0
+
+### Backported versions
+None
 
-* Location of events in archive: events/
-* Id in config: clusterconfig/openshift_machine_api_events
-* Since versions:
-  - 4.12+
+### Changes
+None
 
 
 ## OpenshiftSDNControllerLogs
 
-collects logs from sdn-controller pod in openshift-sdn namespace with following substrings:
-  - "Node %s is not Ready": A node has been set offline for egress IPs because it is reported not ready at API
-  - "Node %s may be offline... retrying": An egress node has failed the egress IP health check once,
-    so it has big chances to be marked as offline soon or, at the very least, there has been a connectivity glitch.
-  - "Node %s is offline": An egress node has failed enough probes to have been marked offline for egress IPs.
-    If it has egress CIDRs assigned, its egress IPs have been moved to other nodes.
-    Indicates issues at either the node or the network between the master and the node.
-  - "Node %s is back online": This indicates that a node has recovered from the condition described
-    at the previous message, by starting succeeding the egress IP health checks.
-    Useful just in case that previous “Node %s is offline” messages are lost,
-    so that we have a clue that there was failure previously.
+Collects logs from sdn-controller pod in openshift-sdn namespace with
```

Comment:
Pod and namespace names are often denoted using monospace font in other gatherers:

```suggestion
Collects logs from `sdn-controller` pod in `openshift-sdn` namespace with
```